ext/Shared/__init__.lua

```diff
@@ -1,227 +1,23 @@
 ---@class CustomLevelLoaderConfig
----@field USE_HTTP boolean
----@field MIRRORS string[]
 ---@field LOGGER_ENABLED boolean
+---@field CLIENT_TIMEOUT number
 Config = require "__shared/Config"
 
+BUNDLE_PREFIX = 'CustomLevels'
```

Comment:
Add this to the config. And you can make the config local now.

Reply:
This shouldnt be configurable, its just a static variable